drivers/soundwire/intel.c

```diff
@@ -974,11 +974,13 @@ static int intel_hw_params(struct snd_pcm_substream *substream,
 static int intel_prepare(struct snd_pcm_substream *substream,
 			 struct snd_soc_dai *dai)
 {
+	struct snd_soc_pcm_runtime *rtd = substream->private_data;
+	struct snd_soc_dai *first_cpu_dai = rtd->cpu_dais[0];
 	struct sdw_cdns *cdns = snd_soc_dai_get_drvdata(dai);
 	struct sdw_intel *sdw = cdns_to_intel(cdns);
 	struct sdw_cdns_dma_data *dma;
 	int ch, dir;
-	int ret;
+	int ret = 0;
 
```

Comment:
nit-pick:  you don't need to set ret = 0;
just do it as: 
````
if (first_cpu_dai != dai)
		return 0;
````

Reply:
> nit-pick: you don't need to set ret = 0;
> just do it as:
> 
> ```
> if (first_cpu_dai != dai)
> 		return 0;
> ```

@RanderWang But we can't return 0 here since other functions are still required when dai != first_cpu_dai. Or you are talking about return 0 after sdw_prepare_stream()? Personally, I prefer to set ret = 0 by default.

Reply:
yes, I mean  do it after sdw_prepare_stream(), just like you did it in intel_tirgger